kola/tests/ignition/filesystem.go

```diff
@@ -253,6 +256,52 @@ func init() {
 		UserData:    swapConfigV2_1,
 		Distros:     []string{"cl"},
 	})
+
+	swapActivation := conf.ContainerLinuxConfig(fmt.Sprintf(`storage:
+  files:
+  - path: /etc/sysctl.d/80-swappiness.conf
+    filesystem: root
+    mode: 0644
+    contents:
+      inline: "vm.swappiness=10"
+
+systemd:
+  units:
+    - name: var-vm-swapfile1.swap
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Turn on swap
+        Requires=create-swapfile.service
+        After=create-swapfile.service
+
+        [Swap]
+        What=%s
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: create-swapfile.service
+      contents: |
+        [Unit]
+        Description=Create a swapfile
+        RequiresMountsFor=/var
+        ConditionPathExists=!%s
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/mkdir -p /var/vm
+        ExecStart=/usr/bin/fallocate -l %s %s
+        ExecStart=/usr/bin/chmod 600 %s
+        ExecStart=/usr/sbin/mkswap %s
+        RemainAfterExit=true`, swapFileName, swapFileName, swapSize, swapFileName, swapFileName, swapFileName))
```

Comment:
How about replacing all `%s` with `%[1]s` and `%[2]s`, so the parameters passed to `Sprintf` would be just `swapFileName, swapSize`? (See [Explicit argument indexes](https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes))

```suggestion
systemd:
  units:
    - name: var-vm-swapfile1.swap
      enabled: true
      contents: |
        [Unit]
        Description=Turn on swap
        Requires=create-swapfile.service
        After=create-swapfile.service

        [Swap]
        What=%[1]s

        [Install]
        WantedBy=multi-user.target
    - name: create-swapfile.service
      contents: |
        [Unit]
        Description=Create a swapfile
        RequiresMountsFor=/var
        ConditionPathExists=!%[1]s

        [Service]
        Type=oneshot
        ExecStart=/usr/bin/mkdir -p /var/vm
        ExecStart=/usr/bin/fallocate -l %[2]s %[1]s
        ExecStart=/usr/bin/chmod 600 %[1]s
        ExecStart=/usr/sbin/mkswap %[1]s
        RemainAfterExit=true`, swapFileName, swapSize))
```

Reply:
Ouh nice, did not about this. Thanks !